CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.3.1 - [](https://github.com/lando/acquia/releases/tag/v1.3.1)
```

Comment:
Use `## {{ UNRELEASED_VERSION }} - [{{ UNRELEASED_DATE }}]({{ UNRELEASED_LINK }})`?  See https://github.com/lando/lando/issues/3710#issuecomment-2101228983.

Reply:
Perfect! Thank you.